src/conways.rs

```diff
@@ -0,0 +1,234 @@
+use std::fmt::Display;
+
+#[derive(Clone, Copy, PartialEq, Debug)]
+enum CellState {
+    Alive,
+    Dead
+}
+
+pub struct Conways {
```

Comment:
I would rename this struct to something more meaningful and less abstract. Given the functions the implementation has, you may rename it to something like `Board`, `ConwaysMap` or `GameMap` (or even `Grid` according to [Wikipedia](https://en.wikipedia.org/wiki/Conway%27s_Game_of_Life)).